tests/e2e/controller_test.go

```diff
@@ -133,10 +172,23 @@ func TestOdhOperator(t *testing.T) {
 
 func TestMain(m *testing.M) {
 	// call flag.Parse() here if TestMain uses flags
-	flag.StringVar(&opNamespace, "operator-namespace",
-		"opendatahub-operator-system", "Namespace where the odh operator is deployed")
-	flag.BoolVar(&skipDeletion, "skip-deletion", false, "skip deletion of the controllers")
+	flag.StringVar(&testOpts.operatorNamespace, "operator-namespace", "opendatahub-operator-system", "Namespace where the odh operator is deployed")
```

Comment:
I haven't changed the `operator-namespace` flag but I'm not sure if the default value can be inherit from the makefile as if I'm not mistaken, the value set in the makefile is not exported as an env var si it would be empty, or am I wrong ?

Reply:
@zdtsw README updated

Reply:
yeah  you are right,
when i have local.env which set OPERATOR_NAMESPACE=openshift-operators
to run
>make e2e-test E2E_TEST_FLAGS="--test-operator-controller=false --test-webhook=false --test-component=dashboard"

equally to 
>go test ./tests/e2e/ -run ^TestOdhOperator -v --operator-namespace=openshift-operators --test-operator-controller=false --test-webhook=false --test-component=dashboard"

default in flag will only be used, when i expeclitly set OPERATOR_NAMESPACE=""